doc/rados/operations/health-checks.rst

```diff
@@ -845,23 +846,24 @@ Data health (pools & placement groups)
 PG_AVAILABILITY
 _______________
 
-Data availability is reduced, meaning that the cluster is unable to
-service potential read or write requests for some data in the cluster.
-Specifically, one or more PGs is in a state that does not allow IO
-requests to be serviced.  Problematic PG states include *peering*,
-*stale*, *incomplete*, and the lack of *active* (if those conditions do not clear
-quickly).
+Data availability is reduced: in other words, the cluster is unable to service
+potential read or write requests for at least some data in the cluster.  More
+precisely, one or more Placement Groups (PGs) are in a state that does not
+allow I/O requests to be serviced. Any of the following conditions can count as
+problematic PG states if they do not clear quickly: *peering*, *stale*,
```

Comment:
Suggest `Any of the following PG states are problematic if they do not clear quickly`